src/slash-bedrock/share/brl-fetch/distros/manjaro

```diff
@@ -0,0 +1,133 @@
+#!/bedrock/libexec/busybox sh
+#
+# Manjaro Linux bootstrap support
+#
+#      This program is free software; you can redistribute it and/or
+#      modify it under the terms of the GNU General Public License
+#      version 2 as published by the Free Software Foundation.
+#
+# Copyright (c) 2016-2019 Daniel Thau <danthau@bedrocklinux.org>
+#
+
+# shellcheck source=src/slash-bedrock/libexec/brl-fetch
+. /bedrock/share/common-code
+trap 'fetch_abort "Unexpected error occurred."' EXIT
+
+check_supported() {
+	true
+}
+
+speed_test_url() {
+	echo "stable/core/${distro_arch}/core.db.tar.gz"
+}
+
+list_mirrors() {
+	# All mirrors:
+	# Server-side filtered list of mirrors:
+	mirror_list_url='https://repo.manjaro.org'
+	wget -qO- "${mirror_list_url}" | list_links |
+	# Remove the header links, and only list http/https
+	tail -n +5 | grep '^http'
+
+}
+
+brl_arch_to_distro() {
+	case "${1}" in
+	"x86_64") echo "x86_64" ;;
+	*) abort "brl does not know how to translate arch \"${1}\" to ${distro} format" ;;
+	esac
+}
+
+list_architectures() {
+	
+	echo "x86_64"
+}
+
+default_release() {
+	echo "rolling"
+}
+
+list_releases() {
+	echo "rolling"
+}
+
+setup_pacman() {
+	# Remove auto architecture detection, as it will confuse things like
+	# i686 on an x86_64 box and armv7hl on a aarch64 box.
+	#
+	# Also remove CheckSpace, which sometimes gets confused by Bedrock's
+	# mount setup.
+	sed 's/^Architecture[ \t]*=.*$//' "${1}/etc/pacman.conf" | \
+		awk -v"a=${distro_arch}" '{print} $0 == "[options]" {print "Architecture = "a}' | \
+		sed 's/^[ \t]*CheckSpace/# &/' \
+		> "${1}/etc/pacman.conf-new"
+	mv "${1}/etc/pacman.conf-new" "${1}/etc/pacman.conf"
+
+	LC_ALL=C chroot "${1}" /usr/bin/update-ca-trust
+	LC_ALL=C chroot "${1}" /usr/bin/pacman-key --init
+	LC_ALL=C chroot "${1}" /usr/bin/pacman-key --populate manjaro archlinux 
```

Comment:
Trailing space.   Search for `make check` in the `Makefile` or `README.md`.

Reply:
No problem -- thanks for the heads up on make check.